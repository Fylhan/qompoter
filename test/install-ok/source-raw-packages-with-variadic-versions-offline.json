{
	"name": "install-ok/qompoter-versionstar-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.*",
		"cp/acme-cp": "v1.*-lib"
	}
}
