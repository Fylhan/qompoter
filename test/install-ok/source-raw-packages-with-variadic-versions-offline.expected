Qompoter
======== install -> create qompoter.lock

* cp/fylhan v1.*
  Downloading sources...
  Selected version: v1.1.10
  done

* cp/acme-cp v1.*-lib
  Downloading lib...
  Selected version: v1.0-lib
  done

done
